File: JAVA/Program/Cloth.java
class Cloth extends Product{
    private String size;
    private String material;
    private String gender;

    public Cloth(){
        this.size = "";
        this.material = "";
        this.gender = "";
    }

    public Cloth(String size, String material, String gender){
        this.size = size;
        this.material = material;
        this.gender = gender;
    }

    public String get_Size() {
        return size;
    }

    public void set_Size(String size) {
        this.size = size;
    }

    public String get_Material() {
        return material;
    }

    public void set_Material(String material) {
        this.material = material;
    }

    public String get_Gender() {
        return gender;
    }

    public void set_Gender(String gender) {
        this.gender = gender;
    }
}

File: PYTHON/Program/Cloth.py
from Product import Product 

class Cloth(Product):
    def __init__(self, size="", material="", gender=""):
        super().__init__()
        self.size = size
        self.material = material
        self.gender = gender

    def get_Size(self):
        return self.size

    def set_Size(self, size):
        self.size = size

    def get_Material(self):
        return self.material

    def set_Material(self, material):
        self.material = material

    def get_Gender(self):
        return self.gender

    def set_Gender(self, gender):
        self.gender = gender